int8 SHORT_PRESS = 1
int8 LONG_PRESS = 2
int8 VERY_LONG_PRESS = 3
int8 IGNORE_PRESS = 4
int8 mode

int8 PRESSED = 0
int8 RELEASED = 1
int8 state
